socialsharing_facebook/lib/AppInfo/Application.php

```diff
@@ -27,27 +27,30 @@
 use OCP\AppFramework\Bootstrap\IBootstrap;
 use OCP\AppFramework\Bootstrap\IRegistrationContext;
 use OCP\EventDispatcher\Event;
-use OCP\EventDispatcher\IEventDispatcher;
+use OCP\EventDispatcher\IEventListener;
 use OCP\Util;
 
-class Application extends App implements IBootstrap {
+/**
+ * @implements IEventListener<\OCA\Files\Event\LoadAdditionalScriptsEvent|\OCA\Files\Event\LoadSidebar>
+ */
+class Application extends App implements IBootstrap, IEventListener {
```

Comment:
Surprising to see an App being an IEventListener but why not :grin:

Reply:
:metal: